config/config.go

```diff
@@ -356,7 +402,29 @@ type StringTransformation struct {
 	Substitution string `yaml:"substitution,omitempty" json:"substitution,omitempty"`
 }
 
-// Load config
+// Load config with the given config file, secret paths and a flag specifying whether analytics credentials must be present.
```

Comment:
I added a chunk of documentation on `Load()` to describe how the config gets loaded. PTAL @theganyo